tests/test_estimators.py

```diff
@@ -17,19 +19,26 @@
 from sknnr._base import IDNeighborsRegressor
 
 
-def get_kneighbor_estimator_instances() -> List[IDNeighborsRegressor]:
+def get_kneighbor_estimator_classes() -> List[Type[IDNeighborsRegressor]]:
     """
-    Return instances of all supported IDNeighborsRegressor estimators.
+    Return classes of all supported IDNeighborsRegressor estimators.
     """
     return [
-        RawKNNRegressor(),
-        EuclideanKNNRegressor(),
-        MahalanobisKNNRegressor(),
-        MSNRegressor(),
-        GNNRegressor(),
+        RawKNNRegressor,
+        EuclideanKNNRegressor,
+        MahalanobisKNNRegressor,
+        MSNRegressor,
+        GNNRegressor,
     ]
 
 
+def get_kneighbor_estimator_instances() -> List[IDNeighborsRegressor]:
```

Comment:
I could go either way too.  I have no problem reading/understanding the inlined comprehension, so I have a slight preference for what you've written here and getting rid of the function.  But please override me if you want.

Reply:
Sounds good, I'm happy with the modified version!

While we're here, I'm feeling guilty about commenting out this test and import... What do you think about using [`pytest.mark.skip`](https://docs.pytest.org/en/7.3.x/reference/reference.html#pytest-mark-skip-ref) instead? The only downside is that pytest reports all the skipped tests when it's run, which adds a bit of clutter to the test output. We could also use [`pytest.mark.xfail`](https://docs.pytest.org/en/7.3.x/reference/reference.html#pytest-mark-xfail) with `run=False`.

Reply:
Marking the test sounds like a good addition.  Since we can provide a reason to `xfail` and we expect this to be fixed at some point, I'm in favor of less cluttered output (since I tend to run tests with `-v` already!).

Reply:
> I'm in favor of less cluttered output (since I tend to run tests with -v already!)

Apparently the `xfail` tests all get reported too, even if they don't run. It does make for a pretty messy log in verbose mode...

![image](https://github.com/lemma-osu/scikit-learn-knn-regression/assets/50475791/bae62c0a-98e9-4b91-8394-79ffffc38adf)

Since the only upside is making our code look a little neater, I'm fine waiting to uncomment them. I'm hoping we can start tackling #21 soon, anyways!